core/overlays/test/pgweb.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: pgweb
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - annotations: null
+    from:
+      kind: DockerImage
+      name: sosedoff/pgweb:0.11.3
+    name: latest
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    service: pgweb
+    app: thoth
+    component: pgweb
+  name: pgweb
+spec:
+  replicas: 1
+  selector:
+    service: pgweb
+  template:
+    metadata:
+      labels:
+        service: pgweb
+    spec:
+      containers:
+      - env:
+        - name: DB_HOST
+          value: postgresql
+        - name: DB_PORT
+          value: "5432"
+        - name: DB_DATABASE
+          valueFrom:
+            secretKeyRef:
+              name: postgresql
+              key: database-name
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              name: postgresql
+              key: database-user
+        - name: DB_PASSWORD
+          secretKeyRef:
+            name: postgresql
+            key: database-password
+        name: pgweb
+        ports:
+          - containerPort: 8081
+            protocol: TCP
+        resources:
+          requests:
+            memory: "256Mi"
+            cpu: "1"
+          limits:
+            memory: "512Mi"
+            cpu: "1"
+        readinessProbe:
+          httpGet:
+            path: '/'
+            port: 8081
+            scheme: HTTP
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          timeoutSeconds: 10
+        livenessProbe:
+          httpGet:
+            path: '/'
+            port: 8081
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 60
+          timeoutSeconds: 10
+  test: false
+  triggers:
+  - type: ConfigChange
```

Comment:
configChange trigger is a little bit tricky, lets see how it behaves...

Reply:
i have removed this for now, as we should first fix this in amun-api or user-api, then we can replicate the behavior in other deployment configs.